semanticdb/metai/src/main/scala/scala/meta/internal/metai/Main.scala

```diff
@@ -47,12 +47,11 @@ final class Main(settings: Settings, reporter: Reporter) {
   }
 
   private def processEntry(file: ClasspathFile): Boolean = {
-    var hasSemanticdb = false
+    val hasSemanticdb = file.path.resolve("META-INF").resolve("semanticdb").isDirectory
```

Comment:
iirc, this did not work when I first implemented this because we did not call `Files.createDirectories` before creating semanticdb files. Keep that in mind for other tools that produce SemanticDB files.

Reply:
Yeah, I had to fix this in metacp.